dockerfiles/run/guice/cassandra-ldap/destination/conf/smtpserver.xml

```diff
@@ -19,6 +19,8 @@
   under the License.
  -->
 
+<!-- Read https://james.apache.org/server/config-smtp.html for further details -->
```

Comment:
dead link?

Reply:
Seems to be instead https://james.apache.org/server/config-smtp-lmtp.html ?